rpc/wrtc_base_channel.go

```diff
@@ -131,17 +131,23 @@ func newBaseChannel(
 				connIDMu.Lock()
 				connID = connInfo.ID
 				connIDMu.Unlock()
-				logger.Infow("Connection state changed",
+				connectionStateChangedLogFields := []interface{}{
```

Comment:
Changes to this file manifest as changing the four info-level logs:
```
2024-11-19T15:21:15.299Z	INFO	rdk.networking	rpc/wrtc_base_channel.go:134	Connection state changed	{"conn_id":"PeerConnection-1732029675298047560","conn_state":"new","conn_local_candidates":[{"FoundAt":"2024-11-19T15:21:15.299000064Z","CandType":"host","IP":"127.0.0.1"},{"FoundAt":"2024-11-19T15:21:15.299000064Z","CandType":"host","IP":"10.1.6.70"}],"conn_remote_candidates":null}
2024-11-19T15:21:15.299Z	INFO	rdk.networking	rpc/wrtc_base_channel.go:134	Connection state changed	{"conn_id":"PeerConnection-1732029675298047560","conn_state":"checking","conn_local_candidates":[{"FoundAt":"2024-11-19T15:21:15.299000064Z","CandType":"host","IP":"127.0.0.1"},{"FoundAt":"2024-11-19T15:21:15.299000064Z","CandType":"host","IP":"172.17.0.1"},{"FoundAt":"2024-11-19T15:21:15.299000064Z","CandType":"host","IP":"100.74.14.74"},{"FoundAt":"2024-11-19T15:21:15.299000064Z","CandType":"host","IP":"10.1.6.70"}],"conn_remote_candidates":null}
2024-11-19T15:21:15.344Z	INFO	rdk.networking	rpc/wrtc_base_channel.go:134	Connection state changed	{"conn_id":"PeerConnection-1732029675298047560","conn_state":"connected","conn_local_candidates":[{"FoundAt":"2024-11-19T15:21:15.344Z","CandType":"server-reflexive","IP":"71.167.222.112"},{"FoundAt":"2024-11-19T15:21:15.344Z","CandType":"host","IP":"127.0.0.1"},{"FoundAt":"2024-11-19T15:21:15.344Z","CandType":"host","IP":"172.17.0.1"},{"FoundAt":"2024-11-19T15:21:15.344Z","CandType":"host","IP":"10.1.6.70"},{"FoundAt":"2024-11-19T15:21:15.344Z","CandType":"host","IP":"100.74.14.74"}],"conn_remote_candidates":[{"FoundAt":"2024-11-19T15:21:15.344Z","CandType":"peer-reflexive","IP":"10.1.6.70"}]}
2024-11-19T15:21:15.345Z	INFO	rdk.networking	rpc/wrtc_base_channel.go:141	Selected candidate pair	{"conn_id":"PeerConnection-1732029675298047560","candidate_pair":"(local) udp4 host 10.1.6.70:38820 <-> (remote) udp4 prflx 10.1.6.70:53043 related :0"}
```
to
```
2024-11-19T15:07:56.165Z	INFO	rdk.networking	rpc/wrtc_base_channel.go:144	Connection establishment succeeded	{"conn_id":"PeerConnection-1732028876106823945","conn_local_candidates":[{"FoundAt":"2024-11-19T15:07:56.164999936Z","CandType":"host","IP":"127.0.0.1"},{"FoundAt":"2024-11-19T15:07:56.164999936Z","CandType":"server-reflexive","IP":"71.167.222.112"},{"FoundAt":"2024-11-19T15:07:56.164999936Z","CandType":"host","IP":"10.1.6.70"},{"FoundAt":"2024-11-19T15:07:56.164999936Z","CandType":"host","IP":"172.17.0.1"},{"FoundAt":"2024-11-19T15:07:56.164999936Z","CandType":"host","IP":"100.74.14.74"}],"conn_remote_candidates":[{"FoundAt":"2024-11-19T15:07:56.164999936Z","CandType":"peer-reflexive","IP":"10.1.6.70"}],"candidate_pair":"(local) udp4 host 10.1.6.70:34019 <-> (remote) udp4 prflx 10.1.6.70:49217 related :0"}
```
The first two are downgraded to debug-level and the latter two are combined into a single log.